harness/determined/common/storage/hdfs.py

```diff
@@ -61,6 +61,7 @@ def download(
         self.client.download(src, dst, overwrite=True)
 
     @util.preserve_random_state
-    def delete(self, tgt: str) -> None:
+    def delete(self, tgt: str, globs: List[str]) -> Dict[str, int]:
```

Comment:
got the go ahead to remove hdfs next release, will do it in a PR which will be up soon